src/devices/machine/myb3k_kbd.h

```diff
@@ -72,8 +76,6 @@ class myb3k_keyboard_device : public device_t
 	void update_modifiers(int y, bool down);
 	virtual void device_timer(emu_timer &timer, device_timer_id id, int param, void *ptr) override;
 
-private:
-
```

Comment:
What's the reason for removing this?  The only base member the derived classes need to access is the protected constructor.